.github/workflows/check-links.yml

```diff
@@ -20,4 +20,3 @@ jobs:
           check-modified-files-only: "yes"
           use-quiet-mode: "yes"
           use-verbose-mode: "yes"
-          base-branch: "main"
```

Comment:
Maybe better to explicitly target master, will be easier to not overlook this when/if moving to main